design/templates.go

```diff
@@ -0,0 +1,46 @@
+// nolint: gochecknoglobals
+package design
+
+import (
+	_ "goa.design/goa/v3/codegen/generator"
+	. "goa.design/goa/v3/dsl"
+)
+
+var _ = API("templates", func() {
+	Title("Templates Service")
+	Description("A service for applying templates to Keboola projects")
+	Server("templates", func() {
+		Host("localhost", func() {
+			URI("http://localhost:8000")
+		})
+	})
+})
```

Comment:
Neviem ci tu `localhost:8000` potrebujeme.
Default hodnota je `localhost:80`, a my aj tak budeme musiet vracat spravnu URL podla stacku.
Lebo inak nebude fungovat `Swager UI` a moznost API vyskusat online.

Reply:
Mozno take nieco? a tym by si mohol vyriesit aj verziu `v1/`, ak teda plati, ze to chceme prefixovat.
Tj. 2 premenne, stack a version:

```
//            Host("production", func() {
//               Description("Production host.")
//               // URIs can be parameterized using {param} notation.
//               URI("https://{version}.goa.design/calc")
//               URI("grpcs://{version}.goa.design")
//
//               // Variable describes a URI variable.
//               Variable("version", String, "API version", func() {
//                   // URI parameters must have a default value and/or an
//                   // enum validation.
//                   Default("v1")
//               })
//           })
```

Reply:
🤔 předpokládal jsem že je to kvůli tomu aby to fungovalo v docker kontejneru, ale zkusím teda jestli to bude fungovat i bez toho

Reply:
myslim, ze je to iba definicia pre openapi? alebo berie HTTP server port z tejto definicie?

Reply:
Jo dobrý funguje. (Ten port nakonec vlastně předávám v argumentu https://github.com/keboola/keboola-as-code/blob/203329058b7285bfa805d651d13857d6c82fff83/Dockerfile-api#L17-L17)